src/Umbraco.Core/Constants-Icons.cs

```diff
@@ -158,5 +158,10 @@ public static class Icons
         ///     System user group icon
         /// </summary>
         public const string UserGroup = "icon-users";
+
+        /// <summary>
+        ///     Webhooks icon
+        /// </summary>
+        public const string Webhooks = "icon-directions-alt";
```

Comment:
Would it be better to keep this in singular form @Zeegaan ?
So `Webhook` like we have User, Folder... although Packages are plural.

Reply:
Yep, sounds correct to me! 😁

Reply:
@Zeegaan does it require VS2022 17.8.0 preview to run `v13/dev` branch?

I tried to enable .NET 8 preview
https://anthonygiretti.com/2023/06/10/net-8-why-net-8-preview-doesnt-show-up-in-visual-studio-2022/

but it seems to return these warnings. 
```
Targeting .NET 8.0 or higher in Visual Studio 2022 17.7 is not supported.
```

so not sure if the static assets are built correct as I get this error on startup:

```
The view '~/umbraco/UmbracoWebsite/Maintenance.cshtml' was not found
```

Reply:
Not using VS (Rider 😍) But yes presumably you'd have to use a preview of VS, when using the dotnet 8 rc-2 🤔

Reply:
https://dotnetthoughts.net/working-with-dotnet8-projects-in-vs2022/

Yes, I think it requires the latest VS2022 preview to run for now:
https://github.com/dotnet/core/issues/8790

Reply:
I just edited in VS 2022, but ran via command line next to it:

```
dotnet build
dotnet run
```